test/util/SpecBase.scala

```diff
@@ -34,5 +35,6 @@ class SpecBase extends AnyWordSpecLike with Matchers with MockitoSugar with Opti
       .configure("metrics.enabled" -> "false")
 
 
-  def fakeRequest(method: String = "", path: String = ""): FakeRequest[AnyContentAsEmpty.type] = FakeRequest(method, path)
+  def fakeRequest(method: String = "", path: String = ""
```

Comment:
Could you please replace "" with by creating a constant named emptyString? If this is to be addressed in further PRs fo this ticket then ignore it.